homeassistant/components/deconz/alarm_control_panel.py

```diff
@@ -154,7 +153,7 @@ def async_update_callback(self, force_update: bool = False) -> None:
     @property
     def state(self) -> str:
         """Return the state of the control panel."""
-        return DECONZ_TO_ALARM_STATE.get(self._device.state, STATE_UNKNOWN)
+        return DECONZ_TO_ALARM_STATE.get(self._device.state, None)
```

Comment:
```suggestion
        return DECONZ_TO_ALARM_STATE.get(self._device.state)
```
Can this key be missing? Use direct access otherwise.

Reply:
At start it could "possibly be anything", thanks for catching None, stupid mistake solving a previous comment...